packages/ui/react-ui-stack/src/components/Section.tsx

```diff
@@ -75,52 +77,57 @@ export const Section = forwardRef<HTMLLIElement, SectionProps>(
               />
             </div>
 
-            {/* Main content. */}
+            {/* Main content */}
             <div role='none' className='flex flex-1 min-is-0 overflow-hidden'>
               {children}
             </div>
-            <DropdownMenu.Root
-              {...{
-                open: optionsMenuOpen,
-                onOpenChange: (nextOpen: boolean) => {
-                  // if (!nextOpen) {
-                  //   suppressNextTooltip.current = true;
-                  // }
-                  return setOptionsMenuOpen(nextOpen);
-                },
-              }}
-            >
-              <DropdownMenu.Trigger asChild>
-                <Button
-                  variant='ghost'
-                  classNames={[
-                    'shrink-0 pli-2 pointer-fine:pli-1 rounded-none',
-                    hoverableControlItem,
-                    hoverableFocusedControls,
-                    hoverableOpenControlItem,
-                    active === 'overlay' && 'invisible',
-                  ]}
-                  // data-testid={testId}
+
+            {/* Menu */}
+            <div role='none' className='flex shink-0'>
+              <div className='p-1'>
```

Comment:
These extra divs are unnecessary — just give `m-1 shrink-0` to the `Button` of `DropdownMenu.Trigger`, since the rest of the children are out-of-flow.